tools/distpartitioning/utils.py

```diff
@@ -512,6 +563,34 @@ def write_dgl_objects(
         dgl.data.utils.save_tensors(orig_eids_file, orig_eids)
 
 
+def get_idranges(names, counts, num_chunks=None):
+    """
+    counts will be a list of numbers of a dictionary.
+    Length is less than or equal to the num_parts variable.
+    """
+    gnid_start = 0
+    gnid_end = gnid_start
+    tid_dict = {}
+    gid_dict = {}
+
+    for idx, typename in enumerate(names):
+        gnid_end += counts[typename]
+        tid_dict[typename] = [[0, counts[typename]]]
+        gid_dict[typename] = np.array([gnid_start, gnid_end]).reshape([1, 2])
+        gnid_start = gnid_end
+
+    return tid_dict, gid_dict
+
+
+def get_typecounts(ntypes, ntype_counts):
```

Comment:
the function name is not intuitive.

Reply:
Changed to 'get_ntype_counts_map', as suggested.